knot-thing/src/main/java/com/cesar/knot_sdk/LogWrapper.kt

```diff
@@ -0,0 +1,28 @@
+package com.cesar.knot_sdk
+
+import android.util.Log
+
+object LogWrapper {
+
+    private const val TAG = "KNOT-SDK"
+    private var logEnable = false
+
+    enum class LogLevel {
+        VERBOSE,
+        DEBUG,
+        INFO,
+        WARN,
+        ERROR
+    }
+
+    fun log(msg : String?, logPriority : LogLevel = LogLevel.DEBUG) {
+
+        if (BuildConfig.DEBUG) when (logPriority) {
```

Comment:
It is expected to use the BuildConfig.DEBUG or the flag logEnable?

Reply:
As discussed, we should use the flag.